tools/analyzer_plugin/test/test_util.dart

```diff
@@ -115,7 +125,7 @@ Future<Map<String, ResolvedUnitResult>> parseAndGetResolvedUnits(Map<String, Str
   for (final path in dartSourcesByPath.keys) {
     final context = collection.contextFor(transformPath(path));
     final result = await context.currentSession.getResolvedUnit(transformPath(path));
-    if (result.errors.isNotEmpty) {
+    if (shouldThrowErrors && result.errors.isNotEmpty) {
```

Comment:
I don't know if ignoring the parse errors has bad side effects. I needed the resolved ast to get the uri information, but it didn't matter if there were static analysis errors in the testcase. Is there a better way I should be doing this?